web/themes/custom/my_awesome_theme/components/00-base/00-defaults/_03-mixins.scss

```diff
@@ -47,6 +47,20 @@
   -webkit-transition: all 0.2s ease;
 }
 
+@mixin title_block{
+  text-align: left;
+  font-family: 'Roboto',sans-serif;
+  font-size: 14px;
+  line-height: 1.4;
+  margin-top: 10px;
+  margin-bottom: 26px;
+  border-bottom: 2px solid var(--td_header_color,var(--td_black,#222222));
+  span{
+    color: #ffffff;
```

Comment:
Why are you don`t use variables for color?
E.g. $color-main: #fff;
color: $color-main;